.gitignore

```diff
@@ -24,7 +24,7 @@ tc/sgx/trusted_worker_manager/enclave/build/
 tc/sgx/trusted_worker_manager/enclave/deps/
 tc/sgx/trusted_worker_manager/enclave_wrapper/enclave_u.c
 tc/sgx/trusted_worker_manager/enclave_wrapper/enclave_u.h
-tc/sgx/common/crypto/verify_ias_report/intel/ias-certificates.cpp
+tc/sgx/common/crypto/verify_ias_report/ias-certificates.cpp
```

Comment:
verify_report module is moved out from crypto and placed in common. so modify this line to 
tc/sgx/common/verify_ias_report/ias-certificates.cpp

Reply:
In this PR, the build actually generates that file as tc/sgx/common/crypto/verify_ias_report/ias-certificates.cpp.

So it would be better to propose another PR addressing this on top of 51ae17072b5fe77e51a6c85f3d1ba66b7a602617.

Reply:
Ok, i will raise a PR to address this change.

Reply:
Thank you 😄  @manju956